src/datasets/non_desktop.md

```diff
@@ -0,0 +1,9 @@
+# Non Desktop Data
+
+## List of Datasets
+
+- [`Day 2-7 Activation`](../../../datasets/non_desktop/day_2_7_activation/reference.md) - Used to calculate the day 2-7 activation metric, a key result in 2020 for the nondesktop products.
+- [`Google Play Store Data`](../../../datasets/non_desktop/google_play_store/reference.md) - Used to understand the acquisition performance for the nondesktop products through the Google Play Store.
+- [`Apple App Store Data`](../../../datasets/non_desktop/apple_app_store/reference.md) - Used to understand the acquisition performance for the nondesktop products through the Apple App Store.
```

Comment:
See above comments about directory structure, but as an interim change we can simplify the links here 
(also have some stylistic suggestions):

```suggestion
- [`Day 2-7 Activation`](./non_desktop/day_2_7_activation/reference.md) - Used to calculate the day 2-7 activation metric, a key result in 2020 for mobile products.
- [`Google Play Store Data`](./non_desktop/google_play_store/reference.md) - Used to understand the acquisition performance for mobile products on the Google Play Store.
- [`Apple App Store Data`](./non_desktop/apple_app_store/reference.md) - Used to understand the acquisition performance for the mobile products on the Apple App Store.
```